.circleci/config.yml

```diff
@@ -45,6 +45,32 @@ anchor_for_install_ktlint: &install_ktlint
 anchor_for_restore_cache: &restore_cache
   key: jars-{{ checksum "build.gradle" }}
 
+anchor_for_install_buf: &install_buf
+  name: Download latest buf
+  command: >
+        get_latest_release() {
+             curl --silent "https://api.github.com/repos/$1/releases" |
+             grep '"tag_name":' |
+             sed -E 's/.*"([^"]+)".*/\1/'
+        }
+
+        BUFBUILD=$(get_latest_release bufbuild/buf)
+
+        echo Using Bufbuild $BUFBUILD
+
+        PROJECT="oppia-android"
+        UNAME_OS=$(shell uname -s)
+        UNAME_ARCH=$(shell uname -m)
+        CACHE_BASE=$(HOME)/.cache/$(PROJECT)
+        CACHE=$(CACHE_BASE)/$(UNAME_OS)/$(UNAME_ARCH)
+        CACHE_BIN=$(CACHE)/bin
```

Comment:
I think you may have to write export before these params and that's why the linter is failing. i.e. 
`export PROJECT="oppia-android"`

Since CircleCI builds are really slow right now, it may be worth trying to run this script locally first to check if its working before pushing it

Reply:
Or could be because of a spacing issue looking at the way the script is rendered in the circleci bash instance . I am not sure but best to take this code into a bash script and run it locally in your machine to fix any errors

Reply:
> I think you may have to write export before these params and that's why the linter is failing. i.e.
> `export PROJECT="oppia-android"`
> 
> Since CircleCI builds are really slow right now, it may be worth trying to run this script locally first to check if its working before pushing it

Got the problem, it's because I use brackets. Checking more locally, getting an error `curl: (23) Failed writing body (0 != 9)`. Will resolve and update you soon.

Reply:
Done

Reply:
@vinitamurthi I think you will have to take a look at it because of following reasons:
1. The Circle CI is still failing.
2. The lines which are mentioned in this comment, they are not available in the code anymore.

Reply:
This part of the file is fine. I think circleCI is failing for other reasons: ktlint had an upgrade